cmd/krel/cmd/ff.go

```diff
@@ -104,18 +104,20 @@ func runFf(opts *ffOptions, rootOpts *rootOptions) error {
 	if err != nil {
 		return errors.Wrap(err, "unable to retrieve current branch")
 	}
-	defer func() {
-		if err := repo.Checkout(currentBranch); err != nil {
-			logrus.Errorf("unable to restore branch %s: %v", currentBranch, err)
-		}
-	}()
-	logrus.Info("Checking out release branch")
-	if err := repo.Checkout(branch); err != nil {
-		return errors.Wrapf(err, "checking out branch %s", branch)
-	}
 
 	if rootOpts.cleanup {
 		defer repo.Cleanup() // nolint: errcheck
+	} else {
```

Comment:
Let's move `currentBranch` into the scope as well:
```suggestion
	} else {
		currentBranch, err := repo.CurrentBranch()
```